profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -80,12 +80,8 @@ class TMGMTPoetryTranslatorPluginController extends TMGMTDefaultTranslatorPlugin
    * and the number of items.
    */
   public function canTranslate(TMGMTTranslator $translator, TMGMTJob $job) {
-    // Check that only one item is being requested.
-    $items = $job->getItems();
-    if (count($items) > 1) {
-      return FALSE;
-    }
     // Check that the item does not have pending requests.
+    $items = $job->getItems();
```

Comment:
In my opinion, we should implement a new TMGMTDefaultTranslatorPlugin for the 'Small Jobs' in the separated feature.